index.js

```diff
@@ -184,14 +184,18 @@ Lexer.prototype = {
   bracketExpression: function(skip){
     skip = skip || 0;
     var start = this.input[skip];
-    assert(start === '(' || start === '{' || start === '[',
-           'The start character should be "(", "{" or "["');
-    var end = ({'(': ')', '{': '}', '[': ']'})[start];
+    var end = characterParser.BRACKETS[start];
+    assert(end, 'The start character should be "(", "{" or "["');
```

Comment:
Strictly speaking, this assert has knowledge that `characterParser.BRACKETS` is an object whose keys are exactly `(`, `{` and `[`. That may lead to trouble if in a year or so we makes changes to the character-parser.

Maybe this message should live in character-parser, either integrated into parseUntil or as its own assertBracketStartChar?